codec_test.go

```diff
@@ -747,6 +747,22 @@ var disassembleResponseStruct = DisassembleResponse{
 	},
 }
 
+// -------- StartDebugging
+
+var startDebuggingRequestString = `{"seq":39,"type":"request","command":"startDebugging","arguments":{"request":"launch","configuration":{"any":true}}}`
+var startDebuggingRequestStruct = StartDebuggingRequest{
+	Request: *newRequest(39, "StartDebugging"),
+	Arguments: StartDebuggingRequestArguments{
+		Request:       "launch",
+		Configuration: map[string]interface{}{"any": true},
+	},
+}
+
+var startDebuggingResponseString = `{"seq":39,"type":"response","request_seq":40,"command":"startDebugging","success":true,"body":{"instructions":[{"address":"someaddr","instruction":"someinstr"}]}}`
```

Comment:
Remove the "body" in this string, since startDebugging responses do not have a body.

Reply:
Oops. Done